src/reactComponents/Tabs.tsx

```diff
@@ -56,7 +56,7 @@ export interface TabsProps {
 }
 
 /** Default copy suffix for tab names. */
-const COPY_SUFFIX = 'Copy';
+const COPY_SUFFIX = ' (copy)';
```

Comment:
This is appended to the module class name in the modal when copying a module. Spaces and parenthesis are not allowed in class names, so this just causes trouble.

<img width="527" height="238" alt="Image" src="https://github.com/user-attachments/assets/eac0dc2b-9fcb-4b99-9627-3346d7e447f8" />